api/v1/flow/flow.proto

```diff
@@ -120,6 +120,11 @@ message Flow {
     // This can be used to extend functionality for other Hubble compatible
     // APIs, or experiment with new functionality without needing to change the public API.
     google.protobuf.Any extensions = 150000;
+
+    // The CiliumNetworkPolicies allowing the egress of the flow.
+    repeated Policy egress_allowed_by = 21001;
```

Comment:
I think you should name this `egress_verdict_by`.

Reply:
Was hoping to keep the existing field names since we were using this in a fork of Cilium and keeping field names (for JSON export) would let us be compatible with our fork.